easybuild/easyconfigs/i/IDG/IDG-1.0.0-gompi-2021b.eb

```diff
@@ -0,0 +1,39 @@
+# Contribution from Imperial College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'IDG'
+version = '1.0.0'
+
+homepage = 'https://www.astron.nl/citt/IDG/'
+description = """Image Domain Gridding (IDG) is a fast method for convolutional resampling 
+(gridding/degridding) of radio astronomical data (visibilities). Direction dependent 
+effects (DDEs) or A-tems can be applied in the gridding process."""
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+
+source_urls = ['https://gitlab.com/astron-idg/idg/-/archive/%(version)s']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+
+checksums = [
+    'b5194e42850d25a34bab8d176986b6c312bbded9a39035e23faad1bc72455263',  # idg-1.0.0.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('CMake', '3.22.1'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('FlexiBLAS', '3.0.4'),
+    ('FFTW', '3.3.10'),
+]
```

Comment:
instead of using `gompi` + `FlexiBLAS` + `FFTW`, just use the `foss` toolchain which includes all of these
```suggestion
```

Reply:
I thought of that as well but then I am also dragging in other modules which I don't need, unless I am mistaken. That was the reason for that solution. That said, I am happy to change it if you think it is a better way forward.